### DISCUSSION TITLE
Empty array of hotkeys

### DISCUSSION THREAD

**Discussion by gavrilikhin-d** [JohannesKlauss/react-hotkeys-hook#1187] [Q&A]
What will happen if I pass empty array for hotkeys?

**Comment by akaguny** [comment]
looks like nothing

https://github.com/JohannesKlauss/react-hotkeys-hook/blob/main/src/useHotkeys.ts#L39
```
const _keys: string = isReadonlyArray(keys) ? keys.join(_options?.splitKey) : keys
```

https://github.com/JohannesKlauss/react-hotkeys-hook/blob/main/src/useHotkeys.ts#L85
```
      parseKeysHookInput(_keys, memoisedOptions?.splitKey).forEach((key) => {
```
https://github.com/JohannesKlauss/react-hotkeys-hook/blob/main/src/parseHotkeys.ts#L38
```
export function parseKeysHookInput(keys: string, splitKey = ','): string[] {
  return keys.split(splitKey)
}
```

console:
```
[].join(',')
''
```
